tests/Nhs.Appointments.Api.Integration/Scenarios/CreateAvailability/GetAvailabilityCreatedEventsFeatureSteps.cs

```diff
@@ -12,9 +12,9 @@ namespace Nhs.Appointments.Api.Integration.Scenarios.CreateAvailability
     [FeatureFile("./Scenarios/CreateAvailability/GetAvailabilityCreatedEvents.feature")]
     public sealed class GetAvailabilityCreatedEventsFeatureSteps : BaseCreateAvailabilityFeatureSteps
     {
-        private HttpResponseMessage _response;
-        private HttpStatusCode _statusCode;
-        private IEnumerable<AvailabilityCreatedEvent> _actualResponse;
+        private new HttpResponseMessage _response;
```

Comment:
if these are hiding members perhaps they should just use the underlying members

Reply:
Yeah it turns out it wasn't even needed on the base class, so I've removed it from there and the new keyword from here